share/doc/INSTALL.md

```diff
@@ -0,0 +1,109 @@
+# VOTCA Installation Guide
+
+## VOTCA structure
+
+VOTCA is currently composed of four major repositories:
+
+* TOOLS - general tools 
+* CSG - topographical classes and course graining functionality
+* CTP - charge transport classes and functionality
+* XTP - exciton and excited states functionality GW-BSE and DFT engine
+
+TOOLS -> CSG -> CTP -> XTP
+
+### Dependency Installation
+#### Ubuntu
+Dependencies for core functionality
+
+    apt-get install make cmake git g++ libexpat-dev libfftw3-dev libgsl-dev libboost-all-dev txt2tags libsqlite3-dev octave gnuplot python-numpy libhdf5-dev graphviz pkg-config psmisc libint2-dev libeigen3-dev libxc-dev libceres-dev
```

Comment:
I am not sure this listing is a good idea a good part e.g libint2 libxc libceres are only for xtp

Reply:
Do you want to split the dependencies section into 3 separate one for csg, ctp and xtp?

Reply:
maybe that is a better idea

Reply:
In my experience, nobody reads documentation anyhow and installing these 2 extra packages might take a min or there isn't much to gain.

Reply:
okay good point. I agree

Reply:
Plus, the default install command has `-DBUILD_XTP=ON` in it.

Reply:
Also see https://github.com/votca/buildutil/commit/09b271f367da2840999f139c58121a7efe364859#diff-0b83f9dedf40d7356e5ca147a077acb4R492, that is there because people just copy paste commands over all the time.

Reply:
lol